randovania/generator/pickup_pool/ammo_pickup.py

```diff
@@ -15,13 +13,15 @@
 def add_ammo_pickups(
     resource_database: ResourceDatabase,
     ammo_configuration: AmmoPickupConfiguration,
-) -> Iterator[PickupEntry]:
+) -> list[PickupEntry]:
```

Comment:
where is this function used? does that not need to be adjusted?

Reply:
No. There's one place and it only cares to iterate over the result, which you can with a list result.